Allow selection of visible layers with '|'.

stahara opened the pull request:
* Toggle (m)onsters, (p)layer, (i)tems, and (c)louds visibility.
* Double-press '|' to invert all (behaves like current '|' command).

wheals commented:
I'm not convinced that there need to be options connected to this, when || to get the old behaviour is so easy.

stahara commented:
I didn't think it would get accepted without it, so it's there.  I can remove it if you like.  It would be much cleaner that way.

MarvinPA commented:
'|' is probably used rarely enough as-is that messing with its functionality is no problem, I agree t hat this would be better with no option.

evktalo commented:
I actually do use the command, and I welcome the new functionality. (Also agree about not needing the option.)

stahara commented:
I also added PR #145 as a squashed and rebased version of this series of commits, since a lot of what was added in the way of configuration (though not functionality) has been again removed.  That PR only modifies 3 files and is much simpler than this series.  You can use whichever you prefer. >8)
